src/gfn/samplers.py

```diff
@@ -84,10 +83,11 @@ def sample_actions(
             log_probs = None
 
         actions = env.actions_from_tensor(actions)
-
         if not save_estimator_outputs:
             estimator_output = None
 
+        assert log_probs is None or log_probs.shape == actions.batch_shape
+        # assert estimator_output is None or estimator_output.shape == actions.batch_shape  TODO: check expected shape
```

Comment:
This also fails. In a test, `estimator_output.shape` was (5, 3), while `actions.batch_shape` was (5, 2)

Reply:
How are you producing these failures?

Reply:
At the moment, I commented the failing asserts, so everything runs fine. However, if you uncomment the asserts and run tests they fail.

The intention for this PR was to leave the logic unchanged, so I should have not introduced a bug, but I might have overseen something

Reply:
OK, thanks!